.gitignore

```diff
@@ -24,3 +24,5 @@ spec/dummy/public/uploads/*
 .rubocop.yml
 
 *.gem
+
+vendor/
```

Comment:
@LeChin Let's remove it from the `.gitignore`, if we remove it it should be from the repo altogether